AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerServiceImpl.kt

```diff
@@ -280,6 +281,19 @@ internal class TestRunnerServiceImpl internal constructor(
         }
     }
 
+    override suspend fun getTestMatrixTestIssues(testMatrix: TestMatrix): List<String> {
+        val steps = testExecutionStore.getTestExecutionSteps(testMatrix)
+        val testIssues = steps.flatMap {
```

Comment:
We should also collect issue type

https://firebase.google.com/docs/test-lab/reference/toolresults/rest/v1beta3/projects.histories.executions.steps#Type

and severity
https://firebase.google.com/docs/test-lab/reference/toolresults/rest/v1beta3/projects.histories.executions.steps#Severity

so we could know how to report it back to AnTS

Reply:
we should ideally also know which step it happened in, so we can attach the issue to the correct work unit.

Reply:
The errorTypes accepted by AnTS are vastly different from what FTL provides. Most of the issues coming from FTL will fall under the "INFRA_ERROR" type. That's why I wasn't sure how useful it'd be to report back. But, I can certainly add these fields to the response.

As for the step, we don't create separate workunits for different shards. So, it going to be very difficult to use the step info to associate failures with that level of granularity.

Reply:
ACK on shards, we can skip step bits.

severity and type still seem useful. We'll likely only want to warn on severe and warning

Reply:
Updated. Please review again. Thanks!